### PR TITLE
LPS-113445 removing form creator name from gretting

### PR DESCRIPTION
**Pull request by linolaoj**

### CODE REVIEW

**Comment by linolaoj** [comment]
ci:test:sf

**Comment by linolaoj** [comment]
ci:test:relevant

**Comment by linolaoj** [comment]
ci:stop

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/resources/notification/form_entry_add_body.soy [hunk 1]
```diff
@@ -54,7 +53,7 @@
 							<div id="introduction" style="background-color: #fff; border-radius: 4px; margin: 0 auto 24px auto; padding: 40px;">
 							<h1 align="center" style="margin-bottom: 24px; margin-top: 0;">{$siteName}</h1>
 
-							<h2 align="center" style="margin-bottom: 24px; margin-top: 0;">Hi, {$authorName}</h2>
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/resources/notification/form_entry_add_body.soy:57]: @linolaoj, please keep the comma. In addition, please visit the class `DDMFormEmailNotificationSender` and remove the `authorName` from there too.

Thanks.

**Comment by linolaoj** [comment]
new pull https://github.com/natocesarrego/liferay-portal/pull/510

**Pull request closed without merging**